css/css-grid/grid-extrinsically-sized-mutations.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid-2/#layout-algorithm">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+<meta name="assert" content="Tests that an extrinsically sized grid responds to various changes to both the grid and its constraints.">
+<style>
+.container {
+  width: 50px;
+}
+.grid {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 50px;
+  height: 50px;
+  outline: 1px solid blue;
+  font: 10px Ahem;
```

Comment:
@sammygill - When writing these tests can you make sure you add a line-height?
See: https://web-platform-tests.org/writing-tests/ahem.html#usage:~:text=computed%20line%2Dheight%20value%20is%20normal

Reply:
Yes, absolutely! I'll fix this along with a similar one I exported recently. Thanks for pointing this out!